src/main/java/com/feelmycode/parabole/controller/CouponController.java

```diff
@@ -37,63 +38,91 @@ public class CouponController {
     private final static int DEFAULT_SIZE = 20;
 
     @PostMapping("/seller/create")
-    public ResponseEntity<CouponCreateResponseDto> addCoupon(
+    public ResponseEntity<ParaboleResponse> addCoupon(
                                     @RequestBody CouponCreateRequestDto dto) {
 
         /** addCoupon, addUserCoupon 이 모두 발생한다. */
-        return ResponseEntity.ok(couponService.addCoupon(dto));
-        // TODO: 에러 처리 해주어야함
+        CouponCreateResponseDto response = couponService.addCoupon(dto);
+        
+        if (response == null) {
+            return ParaboleResponse.CommonResponse(HttpStatus.BAD_REQUEST,
+                false, "제목이 중복되어 쿠폰을 등록하지 못했습니다.", response);
+        }
+        return ParaboleResponse.CommonResponse(HttpStatus.OK,
+            true, "쿠폰 등록 성공", response);
     }
 
     @PostMapping("/seller/giveout")
-    public ResponseEntity<String> assignUserToUserCoupon(@RequestBody Map<String, Object> map) {
+    public ResponseEntity<ParaboleResponse> assignUserToUserCoupon(@RequestBody Map<String, Object> map) {
```

Comment:
couponId와 userId를 받는다면 어차피 PostMapping으로 보니까 RequestParam을 통해서 따로 받는게 낫지 않을까요?
```suggestion
    public ResponseEntity<ParaboleResponse> assignUserToUserCoupon(@RequestParam Long couponId, @RequestParam Long userId) {
```

Reply:
변수 2개는 dto만들어서 던지고,
변수 한개는 전부 @RequestParam 으로 변경했어요 고마워용 ❤️